### PR TITLE
Update codeflare tests environment variables

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
Update codeflare-test's environment variables
Related PR: https://github.com/opendatahub-io/distributed-workloads/pull/186

### CODE REVIEW

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot [hunk 1]
```diff
@@ -10,9 +10,7 @@ Resource          ../../../tests/Resources/Page/DistributedWorkloads/Distributed
 
 *** Variables ***
 ${CODEFLARE_RELEASE_ASSETS}     %{CODEFLARE_RELEASE_ASSETS=https://github.com/opendatahub-io/distributed-workloads/releases/latest/download}
-${NOTEBOOK_IMAGE_STREAM_NAME}   %{NOTEBOOK_IMAGE_STREAM_NAME=s2i-generic-data-science-notebook}
-${NOTEBOOK_ADMIN_NAME}          ${TEST_USER_2.USERNAME}
```
Comment (jiripetrlik) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:14]: It seems not related to the image. Have you tested it?

**Review by sutaakar — APPROVED** [review]

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot (hunk 1 above)
Comment (abhijeet-dhumal) [reply ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:14]: Yes, I have tested this PR locally.. Works as expected !

image

**Review by Srihari1192 — APPROVED** [review]

**Review by ChughShilpa — APPROVED** [review]

**Review by jiripetrlik — APPROVED** [review]
